### PR TITLE
Readd the PK (and rename LightRifle rounds in flight)

### PR DESCRIPTION
**Pull request by TwoDucksOnnaPlane**
## About the PR

Almost fully readded the PK, only missing a couple sprites before it can be mapped.
Renamed the projectiles.
## Why / Balance

It was removed in WizDen [#8301](https://github.com/space-wizards/space-station-14/pull/8301) to "cull excess".
Personally, we can always use more gun, and I renamed the projectiles because it was annoying trying to figure out what shot what.
## Technical details

Nothing but YAML.
## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Readded the PK partially, pending further spriting.
- fix: Fired .30 Rifle rounds are now actually named .30 rifle rounds.

### CODE REVIEW

**Comment by Tmanzxd** [comment]
YAML Linter test fail real. Same with the RSI Validator.

**Comment by TwoDucksOnnaPlane** [comment]
fixing soon™, woe upon me

**Comment by TwoDucksOnnaPlane** [comment]
fixed, 4 line fix in meta.json :trollface:

**Review by Tmanzxd — APPROVED** [review]
These additions look fine. The weapon will need playtested and mapped however then we can come back and rebalance as necessary.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Used by the Vulcan, L6SAW, Mark 1, Ceremonial, and the AKMS. Unsure if there's more.
 - type: entity
   id: BulletLightRifle
-  name: bullet (.20 rifle)
+  name: bullet (.30 rifle)
```
Comment (zelezniciar1) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml:4]: .30 rounds are real!

**Review by zelezniciar1 — APPROVED** [review]
Looks good to me, let's keep an eye on it in the playtest to see if we need to tweak the damage.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml (hunk 1 above)
Comment (TwoDucksOnnaPlane) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml:4]: i have many things to say about the fact that lightrifles shoot .30 but that's a conversation for a different day

**Pull request merged**